srcpkgs/AntiMicroX/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'AntiMicroX'
+pkgname=AntiMicroX
+version=3.1.2
+revision=1
+wrksrc="antimicrox-${version}"
+build_style=cmake
+hostmakedepends="pkg-config extra-cmake-modules itstool gettext qt5-qmake
+ qt5-host-tools"
+makedepends="qt5-tools-devel SDL2-devel libXi-devel libXtst-devel libX11-devel
+ qt5-x11extras-devel"
+short_desc="Map keyboard buttons and mouse controls to a gamepad"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/AntiMicroX/antimicrox"
+distfiles="https://github.com/AntiMicroX/antimicrox/archive/${version}.tar.gz"
+checksum=18539897caed5021003f590cb4baeacf039897c6ff1542697901f573d5a758dc
+
+libAntiMicroX_package() {
```

Comment:
`libantimicrox`. The fact that upstream calls this `libantilib` is unfortunate, would be better to have `libantimicrox`. I suspect they gave it a different name as an easy workaround to avoid conflicting CMake targets, both called `antimicrox`.